src/lib.rs

```diff
@@ -192,38 +211,85 @@ where
     Ok(result)
 }
 
+fn close(event_loop: &PyAny) -> PyResult<()> {
+    event_loop.call_method1(
+        "run_until_complete",
+        (event_loop.call_method0("shutdown_asyncgens")?,),
+    )?;
+
+    // how to do this prior to 3.9?
+    if event_loop.hasattr("shutdown_default_executor")? {
+        event_loop.call_method1(
+            "run_until_complete",
+            (event_loop.call_method0("shutdown_default_executor")?,),
+        )?;
+    }
+
+    event_loop.call_method0("close")?;
+
+    Ok(())
+}
+
 /// Attempt to initialize the Python and Rust event loops
 ///
 /// - Must be called before any other pyo3-asyncio functions.
 /// - Calling `try_init` a second time returns `Ok(())` and does nothing.
 ///   > In future versions this may return an `Err`.
+#[deprecated(since = "0.14.0")]
```

Comment:
It might be a good idea to add `note` to this and other deprecated functions, even if it's pointing to a Github issue or elsewhere.

Reply:
Maybe to the migration guide 👀

Reply:
Yeah, most of the notes should probably at least mention the migration guide. Good catch!